docs/adr/0001-BuildTask.md

```diff
@@ -0,0 +1,21 @@
+# Build Task

```

Comment:
I would not include ADRs in the GitRepo

Reply:
Dropped in e36ea6731206b710e736dd335e93233de553801f